addons/mail/static/src/owl/components/composer_text_input/composer_text_input.js

```diff
@@ -27,553 +17,105 @@ class ComposerTextInput extends Component {
-        });
-        callback(suggestions);
-    }
-
-    /**
-     * Search available commands based on typed characters after the command
-     * trigger character. This should only work when the trigger character is
-     * the 1st character in the contenteditable, because commands should only
-     * work on the whole composer text input content.
-     *
-     * @private
-     * @param {string} keyword
-     * @return {Object[]}
-     */
-    _searchCommandSuggestions(keyword) {
-        const selection = window.getSelection();
-        if (!selection) {
-            return [];
-        }
-        if (!selection.anchorNode) {
-            return [];
-        }
-        if (!selection.anchorNode.parentNode) {
-            return [];
-        }
-
-        /**
-         * @return {DOMNode}
-         */
-        function getAnchorParentFirstChildNotEmptyText() {
-            return Array.prototype.find.call(selection.anchorNode.parentNode.childNodes, childNode =>
-                childNode.nodeType !== 3 || childNode.textContent.trim().length !== 0);
-        }
-
-        if (getAnchorParentFirstChildNotEmptyText() !== selection.anchorNode) {
-            return [];
-        }
-
-        if (this._tribute.current.selectedOffset - 1 !== keyword.length) {
-            return [];
-        }
-
-        const commandList = Object.values(this.env.store.state.commands);
-        const matches = fuzzy.filter(
-            utils.unaccent(keyword),
-            commandList.map(command => command.name));
-        return matches.slice(0, 10).map(match => commandList[match.index]);
-    }
-
-    /**
-     * Search available partner mention suggestions based on typed characters
-     * after the partner mention trigger character.
-     *
-     * @private
-     * @param {string} keyword
-     * @param {function} callback
-     */
-    async _searchPartnerMentionSuggestions(keyword, callback) {
-        this.storeDispatch('searchPartners', { callback, keyword, limit: 10 });
-    }
-
-    /**
-     * Called when contenteditable content should be checked for emptiness.
-     * Useful to enforce html value for "empty".
-     *
-     * @private
+     * @return {boolean}
      */
-    _update() {
-        if (this._editable.textContent.length === 0) {
-            this._editable.innerHTML = EMPTY_HTML;
-        }
-        this._editable.classList.toggle('o-empty', this.isEmpty());
+    isEmpty() {
+        return this._textareaRef.el.value === "";
     }
 
-    //--------------------------------------------------------------------------
-    // Handlers
-    //--------------------------------------------------------------------------
-
     /**
-     * Called when clicking on the contenteditable.
+     * Sets the textarea content.
      *
-     * @private
-     * @param {MouseEvent} ev
+     * @param content
      */
-    _onClickEditable(ev) {
-        this._saveRange();
+    setContent(content) {
+        this._textareaRef.el.value = content;
     }
 
     /**
```

Comment:
missing `// Handlers` section separator above

Reply:
Done (for private too)